setup.cfg

```diff
@@ -21,6 +21,7 @@ install_requires =
     pyyaml
     tqdm
     pytz
+    tzlocal
```

Comment:
Converts the system time into a timezone that pytz can use. I did not know a better way to do this without a package.